packages/state-transition/src/block/processAttestationsAltairPlus.ts

```diff
@@ -28,7 +29,8 @@ const TIMELY_TARGET = 1 << TIMELY_TARGET_FLAG_INDEX;
 const TIMELY_HEAD = 1 << TIMELY_HEAD_FLAG_INDEX;
 const SLOTS_PER_EPOCH_SQRT = intSqrt(SLOTS_PER_EPOCH);
 
-export function processAttestationsAltair(
+export function processAttestationsAltairPlus(
```

Comment:
Should this be called by fork name?
```suggestion
export function processAttestationsDeneb(
```

Reply:
how about `processAttestationsMultiFork`?